docs/quickstart-guide.md

```diff
@@ -0,0 +1,169 @@
+
+_[Learn more about managing images and files](https://www.faecms.com/documentation/tutorials-image_and_files)_
+
+## Fae's Page Generator
+
+Fae has a built in pages module that let's you define content blocks on a page as attributes on a model. This allows you to get content for an about page via `AboutUsPage.instance` instead of dealing with hard-coded IDs.
+
+To use this module, you'll have to follow Fae's convention, but we have a custom genereator to get you started.
+
+**To generate content blocks for an `AboutUsPage`:**
+
+```bash
+rails g fae:page AboutUs hero_image:image headline:string body:text
+```
+
+_[Learn more about content blocks and pages](https://www.faecms.com/documentation/topics-pages)_
+
+**Finally let's add the scaffolded objects to the database:**
+
+```bash
+rails db:migrate
+```
+
+_Use `rake db:migrate` with Rails 4_
+
+## Customizing the Forms
+
+With those four commands we are now able to manage those three objects in our CMS. There are some things we can do now to make these forms easier to work with.
+
+### Article Categories
+
+Go to http://localhost:3000/admin/article_categories and add/update a couple categories. You'll notice that Fae picked up the `position` attribute and automatically made the categories sortable in the index view.
+
+### Articles
+
+Now let's go to create a new article at http://localhost:3000/admin/articles/new. This form already does the job, but let's make it better. Fae's generators create files in your application so you can customize this form as we please.
+
+Open `app/views/admin/articles/_form.html.slim` and focus on the `main.content` section where the form fields are:
+
+```ruby
+main.content
+  = fae_input f, :title
+  = fae_input f, :slug
+  = fae_input f, :introduction
+  = fae_input f, :body
+  = fae_input f, :date
+
+  = fae_association f, :article_category
+
+  = fae_image_form f, :hero_image
+  = fae_file_form f, :pdf
+```
+
+
+**Automatically generate the `slug` from content in the `title`.**
+
+Fae's form helpers offer many options to help you customize. Let's add a class to the `title` field to trigger Fae's slugger feature:
+
+```ruby
+= fae_input f, :title, input_class: 'slugger'
+```
+
+_[Learn more about Fae's slugger](https://www.faecms.com/documentation/features-slugger)_
+
+**Add a markdown editor to `body`.**
+
+```ruby
+= fae_input f, :body, markdown: true
+```
+
+**Add a datepicker to `date`.**
+
+Fae also has a number of other form helpers that provide a variety of UI elements.
+
+```ruby
+fae_datepicker f, :date
+```
+
+_[See documentation on all form helpers](https://www.faecms.com/documentation/helpers-form_helpers)_
+
+## Form Validations
+
+Fae's forms will honor any standard validation rules defined in the model. Fae even has some [helpers for common validation scenerios](https://www.faecms.com/documentation/topics-models#validation).
+
+To add form validations to our articles, we just need to define them in `app/models/article.rb`
```

Comment:
...article.rb`:` This is only because every preceding sentence before the code blocks has a period or colon.